### PR TITLE
feat(content): Add new uninhabited landable planets to existing systems in Human space

### PR DESCRIPTION
**Pull request by Quantumshark**
**Content**

This PR does help somewhat with the problems described in issue #4628 

## Summary
Adds four new uninhabited landable planets to existing systems in Human space - specifically, in Gacrux, Alpheratz and Nocte.

This both adds rewards to exploration, in the form of interesting things that can be found (sort of like #8274 did, though in this case they can be found while still flying a human ship), and provides safeish stopovers that pirate players can use without needing to give the spaceport authorities a cut of their hard-earned plunder.
And now there's actually a reason to check out at least some of those uninhabited dead end systems.

## Screenshots
image
In pink are the three existing uninhabited landables I remembered when I made this image: Hope, Watcher and Mutiny. In yellow are the four new ones: Asphodel, Neriene, New Kola and Asterus.

## Testing Done
None whatsoever.

## Save File
This save file can be used to test these changes:
Any in human space, pretty much.

### CODE REVIEW

File: data/map planets.txt [hunk 1]
```diff
@@ -3061,6 +3080,12 @@ planet "New Kansas"
 		threshold 2500
 		fleet "Small Militia" 22
 
+planet "New Kola"
+	attributes uninhabited
+	landscape land/badlands7
+	description `Although from outside New Kola appears to be a gas giant, its thick, dense atmosphere obscures the rocky surface of a huge super-earth. Even if it were made of breathable gases, the pressure at the surface is beyond what humans can tolerate - only at the peaks of the highest mountains can you safely step outside wearing only oxygen gear, and even then the crushing gravity would cause serious problems if you stayed for long.`
```
Comment (MasterOfGrey) [inline data/map planets.txt:3086]: If the air pressure is broadly tolerable, so is the gravity.

Comment (Quantumshark) [reply data/map planets.txt:3086]: I'll admit I'm not particularly well-versed in planetary geography, but what I'm trying to say here is that the air pressure is only tolerable on high mountain peaks, not at ground level.
To the best of my knowledge though, there's quite a large degree of variability in atmospheric pressure relative to gravity - see Earth vs Venus, or Titan vs Ganymede.

File: data/map planets.txt [hunk 2]
```diff
@@ -249,12 +249,24 @@ planet "Ashy Reach"
 	"required reputation" 15
 	security 0.1
 
+planet Asphodel
+	attributes uninhabited
+	landscape land/badlands3
+	description `Despite being nominally in the habitable zone, this planet's surface is continually scorched by flares from the system's stars, steadily blasting away its atmosphere. Although some small research stations are present in orbit, providing valuable insights into the nature and behavior of flare stars, the researchers generally avoid descending to the planet's surface unnecessarily.`
```
Comment (bene-dictator) [inline data/map planets.txt:255]:
```suggestion
	description `Despite being nominally in the habitable zone, this planet's surface is continually scorched by Gacrux and its solar flares, steadily blasting away its atmosphere. The Republic has put some small research stations in orbit around Asphodel, and while they provide valuable insights into the nature and behavior of flare stars, the researchers generally avoid descending to the planet's surface unnecessarily. None of the stations appear to have the facilities to service your ship, however.`
```
Not sure if Gacrux has multiple stars in real life, from a search it says it's debated whether it has a white dwarf companion to the confirmed red giant.

Maybe instead of the Republic, could be the Deep that funds the stations, or Bounty / Heartland / some university from Chiron or the Paradise Planets.

File: data/map planets.txt [hunk 3]
```diff
@@ -249,12 +249,24 @@ planet "Ashy Reach"
...
 	spaceport `The spaceport on Aspiration is a spaceport in the same way that an airport at a regional town is an airport. It has the most minimal facilities to handle a ship landing without disturbance to the nearby town, and that is it. There are no shops or stalls. Aspiration is not on any trade routes and it does not expect visitors. A lone freighter is parked permanently on the outskirts.`
 
+planet Asterus
+	attributes moon uninhabited
+	landscape land/enceladus_2
+	description `The seas beneath the frozen surface of this ice moon are home to a thriving ecosystem full of species entirely unlike any observed on more conventionally habitable worlds. The best known of these is a species of large fish-like creatures with flowing bioluminescent fins that glow in ethereal pinks and purples, the beauty of which prompted many citizens of the Paradise Worlds to demand that the Republic designate this moon as a nature preserve.`
```
Comment (bene-dictator) [inline data/map planets.txt:267]:
```suggestion
	description `The seas beneath the frozen surface of this ice moon are home to a thriving ecosystem full of species entirely unlike any observed on other, more habitable worlds. The best known of these is a species of large fish-like creatures with flowing bioluminescent fins that glow in ethereal pinks and purples, the beauty of which prompted many citizens of the Paradise Worlds to demand that Parliament designate the entire moon as a nature preserve in the 2900s.`
```
I felt like giving some sort of date, picked an arbitrary century.

File: data/map planets.txt [hunk 4]
```diff
@@ -2912,6 +2924,13 @@ planet Nepheritis
 	description `	While these diamonds are unfortunately not economical to collect with current human technology, if you were so inclined you could perhaps make an attempt at catching one as it fell, deep within the atmosphere where they are largest.`
 	government Uninhabited
 
+planet Neriene
+	attributes uninhabited
+	landscape land/canyon03
+	description `Other than its lack of the iconic red color, this planet is almost eerily similar in conditions to Humanity's second planet, Mars. The thin atmosphere of carbon dioxide, the polar ice caps, and the echoes of dried waterways across the surface all tell the tragic tale of the same fate that befell Mars in aeons past.`
+	description `But although, when they were confined to their home system, humanity eventually triumphed over the adverse conditions and made a home on the Red Planet, this near-identical world has been left totally barren, a mere afterthought to the now interstellar Human civilisation.`
```
Comment (bene-dictator) [inline data/map planets.txt:2931]:
```suggestion
	description `While lacking its iconic ferrous-red color, this planet is almost eerily similar in conditions to the second planet humanity set foot on, Mars. The thin atmosphere of carbon dioxide, the polar ice caps, and the echoes of dried waterways across the surface all tell the tragic tale of the same fate that befell Mars in aeons past.`
	description `	Even though humanity eventually triumphed over the adverse conditions and made a home on the "Red Planet," this near-identical world has been left totally barren, a mere afterthought in the now interstellar human civilisation.`
```

File: data/map planets.txt (hunk 1 above)
Comment (bene-dictator) [inline data/map planets.txt:3086]:
```suggestion
	description `At a distance, New Kola appears to be a gas giant; yet upon approach you notices that the its thick, dense atmosphere obscures the rocky surface of a huge super-earth. Even if it were made of breathable gases, the pressure at the surface is beyond what humans can tolerate - only at the peaks of the highest mountains can you safely step outside wearing only a breathing apparatus, and even then the crushing gravity would cause serious problems if you stayed for long.`
```

**Review by bene-dictator — CHANGES_REQUESTED** [review]
Nice! ~~I may do the same myself. Later.~~

File: data/map planets.txt (hunk 2 above)
Comment (Quantumshark) [reply data/map planets.txt:255]: I based this off the stars that are there in-game (a binary of a k8 and an m5), not what's there in real life.

Comment (Quantumshark) [reply data/map planets.txt:255]: > Maybe instead of the Republic, could be the Deep that funds the stations, or Bounty / Heartland / some university from Chiron or the Paradise Planets.

I do like the idea of mentioning one of the universities on Chiron here, and that does seem appropriate.

Comment (bene-dictator) [reply data/map planets.txt:255]: Yeah fair. ~~I mean it could get changed, or...~~
```suggestion
	description `Despite being nominally in the habitable zone, this planet's surface is continually scorched by the stars of Gacrux and their solar flares, steadily blasting away its atmosphere. The Republic has put some small research stations in orbit around Asphodel, and while they provide valuable insights into the nature and behavior of flare stars, the researchers generally avoid descending to the planet's surface unnecessarily. None of the stations appear to have the facilities to service your ship, however.`
```

File: data/map planets.txt (hunk 1 above)
Comment (MasterOfGrey) [reply data/map planets.txt:3086]: Air pressure is proportional to gravity. If there is a place where you can land and tolerate the air pressure pretty well (without also needing protective gear because it's corrosive or whatnot), then _at that place_ you can also tolerate the gravity pretty well.

Titan has similar pressure to earth because it's atmosphere is made up of much heavier stuff, and is super cold.
Venus atmosphere has much greater pressure than earth because it's made of of much heavier stuff, and is super corrosive.

If you can safely step outside with _just_ oxygen gear at a particular level in the atmosphere, you can also tolerate the gravity at the same level as well.

Comment (Quantumshark) [reply data/map planets.txt:3086]: > Air pressure is proportional to gravity. If there is a place where you can land and tolerate the air pressure pretty well (without also needing protective gear because it's corrosive or whatnot), then at that place you can also tolerate the gravity pretty well.

I don't think that's right. At the top of Mount Everest, the atmospheric pressure is about one third of that at sea level, and the gravity is almost exactly the same as at sea level. To take a more extreme example, the gravitational pull experienced at the altitude of low earth orbit is less than 10% less than at sea level, but the atmospheric pressure is zero.
As far as I can tell, atmospheric pressure is the result of the weight (i.e. mass times gravity), of all the atmosphere above a particular point, so it would be increased by increasing the mass of the atmosphere or by increasing the strength of gravity. But as you move up, an increasing portion of the atmosphere is now below you rather than above, and hence isn't weighing down on the atmosphere where you are.

Comment (MasterOfGrey) [reply data/map planets.txt:3086]: And at the top of Everest you need more than just oxygen gear because it's cold enough to kill you, and quite quickly.

> If you can safely step outside with **_JUST_** oxygen gear at a particular level in the atmosphere, you can also tolerate the gravity at the same level as well.

Comment (Quantumshark) [reply data/map planets.txt:3086]: Sure, but a thicker atmosphere would capture more heat. I imagine all other things being equal, you'd expect that temperatures at the altitude where the atmospheric pressure is about 1atm would be about the same whether that's ground level or it's the top of a mountain and there's ten kilometers more atmosphere underneath you. Also, this atmosphere has a different composition to Earth's, which would also have effects on the surface temperature.

Comment (MasterOfGrey) [reply data/map planets.txt:3086]: > I imagine all other things being equal, you'd expect that temperatures at the altitude where the atmospheric pressure is about 1atm would be about the same whether that's ground level or it's the top of a mountain and there's ten kilometers more atmosphere underneath you.
You would imagine wrongly. For example, Jupiter's atmosphere is -108C (-163F) where the pressure is 1atm. Saturn's is -138C. Uranus -197C.
Even Venus, which has a cloud layer at approximately 1atm that would be tolerable to humans is 30C compared to Earth's average 15C. And that particular layer is relatively nice gasses like nitrogen and carbon dioxide.

Temperature, composition, pressure, and atmosphere volume are all inputs into a multivariable function where we survive in a very narrow window.

If you can step outside in your everyday cotton flight suit or whatever, and JUST use oxygen gear, you can tolerate the gravity.

Comment (Quantumshark) [reply data/map planets.txt:3086]: > You would imagine wrongly. For example, Jupiter's atmosphere is -108C (-163F) where the pressure is 1atm. Saturn's is -138C. Uranus -197C.

I did say "All other things being equal". Jupiter, Saturn and Uranus orbit far outside the habitable zone, and recieve far less sunlight. That's not "all other things being equal", because solar insolation is an "other thing" and is not equal.

Conversely, Venus is closer to the sun and recieves more sunlight, though I'm a little surprised by how little difference that makes at the 1atm layer.

> Temperature, composition, pressure, and atmosphere volume are all inputs into a multivariable function where we survive in a very narrow window.

You're correct that these are all inputs into a function, and that we can only survive in a narrow window, but that doesn't mean only a narrow range of combinations of conditions is possible. Gravity is determined by planetary mass and density (or radius and density, or mass and radius - any two determine the third), largely independent of the atmosphere. Atmospheric pressure is then determined based on that and on the total amount of atmosphere present, which, as I pointed out, can vary massively for a given planetary mass. (plot Titan, Ganymede, Earth, Venus, and Mars on a graph, and what you certainly don't get is a nice line). Temperature then can be varied with no effect on gravity and little effect on total atmosphere (as long as gases don't start freezing out) based on the amount of light recieved from the star. 
With all that in mind, I don't think the combination of characteristics here is impossible - It's just difficult to produce a concrete proof because our solar system lacks any terrestrial planets larger than earth. I think it's pretty plausible that a super-earth type terrestrial planet, with an atmosphere somewhere between Earth's and Venus', would be pretty much exactly as I describe New Kola.

File: data/map planets.txt [hunk 5]
```diff
@@ -2912,6 +2924,13 @@ planet Nepheritis
 	description `	While these diamonds are unfortunately not economical to collect with current human technology, if you were so inclined you could perhaps make an attempt at catching one as it fell, deep within the atmosphere where they are largest.`
 	government Uninhabited
 
+planet Neriene
+	attributes uninhabited
+	landscape land/canyon03
+	description `While lacking its iconic ferrous-red color, this planet is almost eerily similar in conditions to the second planet humanity set foot on, Mars. The thin atmosphere of carbon dioxide, the polar ice caps, and the echoes of dried waterways across the surface all tell the tragic tale of the same fate that befell Mars in aeons past.`
+	description `	Even though humanity eventually triumphed over the adverse conditions and made a home on the Red Planet, this near-identical world has been left totally barren, a mere afterthought in the now interstellar human civilisation.`
```
Comment (tibetiroka) [inline data/map planets.txt:2931]:
```suggestion
	description `	Even though humanity eventually triumphed over the adverse conditions and made a home on the Red Planet, this near-identical world has been left totally barren, a mere afterthought in the now interstellar human civilization.`
```

**Review by tibetiroka — CHANGES_REQUESTED** [review]

**Review by EjoThims — CHANGES_REQUESTED** [review]

File: data/map planets.txt [hunk 6]
```diff
@@ -249,12 +249,24 @@ planet "Ashy Reach"
 	"required reputation" 15
 	security 0.1
 
+planet Asphodel
+	attributes uninhabited
+	landscape land/badlands3
+	description `Despite being nominally in the habitable zone, this planet's surface is continually scorched by the stars of Gacrux and their solar flares, steadily blasting away its atmosphere. A joint venture of three major universities from Chiron has placed some small research stations in orbit around Asphodel, and while they provide valuable insights into the nature and behavior of flare stars, the researchers generally avoid descending to the planet's surface unnecessarily. None of the stations appear to have the facilities to service your ship, however.`
```
[the remark this answers is not in the document]
Comment (Quantumshark) [reply data/map planets.txt:255]: I think technically it's not necessary, but it's not wrong either and I think it reads better with the comma than without.

File: data/map planets.txt (hunk 1 above)
Comment (MasterOfGrey) [reply data/map planets.txt]: For the love of god I don't have the time or the crayons for this, but at the _very least_ you need some protection from wind chill to otherwise be there with only breathing gear.

Comment (Hecter94) [reply data/map planets.txt]: Insulting people's intelligence isn't necessary, regardless of how much in the right you feel you are.

Comment (roadrunner56) [reply data/map planets.txt]: Gravity and wind chill are two separate things though. If you want to argue that it would be *too cold* to step outside with only oxygen gear, that's one thing, but that's different from your previous argument that the air pressure would necessarily be intolerable.

Comment (MasterOfGrey) [reply data/map planets.txt]: It's a combination of things. With the given combination as it stands, either the temp is cold and wind chill will freeze you, or the temp is warm and the wind will suck the moisture from you and you'll die of dehydration quicker than the gravity will be an issue.
And even that possibility is falling into an extremely narrow range of only hypothetically plausible possibilities.

> air pressure would necessarily be intolerable.

And I didn't say it would *necessarily* be intolerable, I said that *either* the pressure would be intolerable, or the atmosphere would kill you other ways - which remains consistent with my point.

**Comment by bene-dictator** [comment]
Added new landing pictures in Quantumshark/endless-sky#3. I'll do the copyright and `@2x` later.

File: data/map planets.txt [hunk 7]
```diff
@@ -3061,6 +3080,12 @@ planet "New Kansas"
 		threshold 2500
 		fleet "Small Militia" 22
 
+planet "New Kola"
+	attributes uninhabited
+	landscape land/haze2
+	description `At a distance, New Kola appears to be a gas giant; yet upon approach you notice that the thick, dense atmosphere obscures the rocky surface of a huge super-earth. Even if it were made of breathable gases, the pressure at the surface is beyond what humans can tolerate - only at the peaks of the highest mountains can you safely step outside wearing only a breathing apparatus, and even then the crushing gravity would cause serious problems if you stayed for long.`
```
Comment (ziproot) [inline data/map planets.txt:3086]:
```suggestion
	description `At a distance, New Kola appears to be a gas giant; yet upon approach you notice that the thick, dense atmosphere obscures the rocky surface of a huge super-earth. Even if it were made of breathable gases, the pressure at the surface is beyond what humans can tolerate - only at the peaks of the highest mountains can you safely step outside wearing only a breathing apparatus, and even then the pressure and crushing gravity would cause serious problems if you stayed for long.`
```

There are three things here.

1. Gravity is mentioned to be “crushing.” Super earths are generally about 2-10x the mass and around the same radius as the Earth, so I can assume about 2-10x the gravity. I will assume it is about 5x as much, which is beyond the range of what humans can tolerate.

2. Surface pressure is greater than Earth. However, there are **many** factors that control surface pressure, with gravity only being one of them. Atmospheric composition, atmospheric thickness, and altitude are the main ones, however, temperature also controls pressure due to the ideal gas law. As temperature increases, pressure decreases. Given Mars gravity and atmospheric composition, a pressure of 35 atm on the surface and a surface temperature of 95°C leads to about 22.0 atm, or 15.5 atm for Earth and 13.2 atm for Venus ([source](https://www.mide.com/interplanetary-air-pressure-at-altitude-calculator)). Higher gravity leads to a slower decrease in pressure, whereas a more dense atmosphere leads to a faster decrease in pressure, so Mars has a slower decrease in pressure as its atmosphere is so thin. As a super Earth would have higher gravity and a more dense atmosphere, you could argue that the density somewhat counters the higher gravity. The higher gravity itself would lead to a decrease of 1/5 as much with Earth atmospheric density, leading to about 31 atm. The difference in pressure change between Venus and Earth means that with Venus composition, pressure would be 26 atm. So, pressure would probably be something like 28 atm, which the same as the experienced at around 280m underwater. With breathing equipment, it *is* possible to go 280m underwater, so I don’t think pressure like this is too bad for the constraints given. 

4. For temperature, depending on the atmospheric composition, it can actually increase with elevation. On Earth, this happens in the stratosphere, due to the presence of ozone. The implication is that there is no oxygen and therefore no ozone layer, meaning no temperature inversion causing the temperatures to increase at some elevation, so the temperatures will just continuously decrease until the atmosphere becomes so thin that temperatures increase. Using the dry adiabatic lapse rate, there would be pleasant temperatures at around 8 km up. However, that’s assuming 90°C temperatures. The planet appears to be in the habitable zone, so some extra greenhouse gases leading to warmer temperatures isn’t that hard to come up with.

Basically, the pressure would be high, but the criteria is tolerable considering you are using specialized breathing equipment (“oxygen gear”), and while 35 atm is not tolerable, 28 atm can be, as the limit is around 30 atm. Of course, you still probably wouldn’t want to stay very long, but you can step outside. I would probably modify the final sentence slightly, but I don’t think that is necessary.

Comment (Quantumshark) [reply data/map planets.txt:3086]: > Gravity is mentioned to be “crushing.” Super earths are generally about 2-10x the mass and around the same radius as the Earth, so I can assume about 2-10x the gravity. I will assume it is about 5x as much, which is beyond the range of what humans can tolerate.

I thought large terrestrial planets had similar densities to earth, but larger masses and radii, and higher surface gravity - after all, rock isn't compressible like gas is, so only changing the composition would change the density. Of course, mass is proportional to the cube of radius (for a given density), so with 2x the radius you'd have 8x the mass, and hence 4x the surface gravity. That's probably just a little bigger than New Kola is, and it might be made of rock with a slightly lower average density than Earth, so maybe surface gravity is something like 3.5 times Earth gravity.

> So, pressure would probably be something like 28 atm, which the same as the experienced at around 280m underwater. With breathing equipment, it is possible to go 280m underwater, so I don’t think pressure like this is too bad for the constraints given.

I didn't realise humans could survive anything like that much atmospheric pressure - I'd have guessed the upper limit was more like 10atm. Still, I think you're right in saying that the higher gravity would make atmospheric pressure drop off more quickly with altitude, so getting from 30atm at ground level down to 1-2 atm at the peaks of the highest mountains doesn't seem too absurd? Based on the numbers I can find, that might require mountains about 20km high, which is slightly smaller than Olympus Mons, so not completely implausible?

Comment (ziproot) [reply data/map planets.txt:3086]: > . Still, I think you're right in saying that the higher gravity would make atmospheric pressure drop off more quickly with altitude

Less quickly, as the surface pressure would be more from gravity and less from a dense atmosphere. Compare Earth to Jupiter with all values being the same. For 3x as much gravity, pressure drops at 1/3 the rate.

> I didn't realise humans could survive anything like that much atmospheric pressure - I'd have guessed the upper limit was more like 10atm

Divers can go more than 100m underwater, though very few have gone more than 300m underwater, so 30 atm is basically the cutoff. The main issue is not inhaling the wrong oxygen pressure, but your oxygen gear would likely have the right concentrations to counter this.

> Based on the numbers I can find, that might require mountains about 20km high, which is slightly smaller than Olympus Mons, so not completely implausible?

30 atm would lead to 17.7-21.1 atm for Earth/Venus with 3x gravity, a 20000m altitude, and surface temperatures of 90°C, so around 18-19 atm. That isn’t close to 1 atm due to the stronger gravity, but it is close to 15 atm, which is how much the ship can definitely withstand, so 20km seems reasonable to me with close to 30 atm pressure.

Comment (warp-core) [reply data/map planets.txt:3086]: > Of course, mass is proportional to the cube of radius (for a given density), so with 2x the radius you'd have 8x the mass, and hence 4x the surface gravity.

Because you're twice as far away from the centre, the surface gravity will only be twice.

Earth surface gravitational force = G \* M_e / (R_e)^2
Where:
- G is the gravitational constant
- M_e is Earth's mass.
- R_e is Earth's radius.

If we consider a planet with the same density as Earth with twice the radius, it's mass will indeed be 8 times Earth's mass.
So, gravitational force on this planet's surface = G \* 8 \* M_e / (2 \* R_e)^2 = (8 / 4) \* G \* M_e / (R_e)^2 = 2 * G \* M_e / (R_e)^2

I'm not doing anymore maths right now, but I do intend to apply some realism to the discussions here at some point (unless someone else does it first.)

Comment (Amazinite) [reply data/map planets.txt:3086]: > Based on the numbers I can find, that might require mountains about 20km high, which is slightly smaller than Olympus Mons, so not completely implausible?

Keep in mind that Olympus Mons is taller than mountains on Earth in part because Mars has lower gravity. Lower gravity allows mountains to grow taller, as it takes more height for them to collapse under their own mass. A planet with a higher gravity will be likely have smaller mountains.

> As a super Earth would have higher gravity and a more dense atmosphere

Need a super earth necessarily have a more dense atmosphere?

Comment (ziproot) [reply data/map planets.txt:3086]: A Super Earth only means it has a mass of 2-10x that of Earth, and can also mean a radius of around 0.8-1.2x that of Earth.

**Review by roadrunner56 — APPROVED** [review]
Debate about planet physics aside, the grammar looks good.

File: copyright [hunk 8]
```diff
@@ -212,6 +212,30 @@ License: public-domain
  for the MESSENGER mission. Because it was produced for
  a NASA mission, it is in the public domain.
 
+Files:
+ images/land/haze0*
+Copyright: Izaac Hewitt-Lawrence
+License: Unsplash License
+ Taken from unsplash.com and cropped.
```
Comment (bene-dictator) [inline copyright:219]:
```suggestion
Files:
 images/land/haze0*
Copyright: Izaac Hewitt-Lawrence
License: Unsplash
 Taken from unsplash.com and cropped. This image was
 uploaded to unsplash.com after June 2017, so it is subject
 to the Unsplash License. It was also uploaded after February
 2018, so it is subject to an additional restriction limiting
 the sale of unaltered copies.


```

File: copyright [hunk 9]
```diff
@@ -212,6 +212,30 @@ License: public-domain
...
+ images/land/haze1*
+Copyright: Tobias Rademacher
+License: Unsplash License
+ Taken from unsplash.com and cropped.
+
+Files:
+ images/land/haze2*
+Copyright: Marek Piwnicki
+License: Unsplash License
+ Taken from unsplash.com and cropped.
```
Comment (bene-dictator) [inline copyright:231]:
```suggestion
Files:
 images/land/haze2*
Copyright: Marek Piwnicki
License: Unsplash
 Taken from unsplash.com and cropped. This image was
 uploaded to unsplash.com after June 2017, so it is subject
 to the Unsplash License. It was also uploaded after February
 2018, so it is subject to an additional restriction limiting
 the sale of unaltered copies.
 
```

File: copyright [hunk 10]
```diff
@@ -212,6 +212,30 @@ License: public-domain
...
+
+Files:
+ images/land/snow23*
+Copyright: Daniel Frank
+License: Unsplash License
+ Taken from unsplash.com and cropped.
+
```
Comment (bene-dictator) [inline copyright:238]:
```suggestion
Files:
 images/land/snow23*
Copyright: Daniel Frank
License: Unsplash
 Taken from unsplash.com and cropped. This image was
 uploaded to unsplash.com after June 2017, so it is subject
 to the Unsplash License. It was also uploaded after February
 2018, so it is subject to an additional restriction limiting
 the sale of unaltered copies.

```

File: copyright [hunk 11]
```diff
@@ -212,6 +212,30 @@ License: public-domain
...
+Copyright: Izaac Hewitt-Lawrence
+License: Unsplash License
+ Taken from unsplash.com and cropped.
+
+Files:
+ images/land/haze1*
+Copyright: Tobias Rademacher
+License: Unsplash License
+ Taken from unsplash.com and cropped.
+
```
Comment (bene-dictator) [inline copyright:226]:
```suggestion
Files:
 images/land/haze1*
Copyright: Tobias Rademacher
License: Unsplash
 Taken from unsplash.com and cropped. This image was
 uploaded to unsplash.com after June 2017, so it is subject
 to the Unsplash License. It was also uploaded after February
 2018, so it is subject to an additional restriction limiting
 the sale of unaltered copies.

```

**Review by bene-dictator — CHANGES_REQUESTED** [review]
New copyright guidelines make my lazy copyrighting skills apparent.

**Comment by bene-dictator** [comment]
Oh yeah, there's one other error with the copyright the CI/CD found, but I don't really know what it means.

> Error: More generic patterns should precede specific ones: images/land/snow23* and images/land/snow2*

File: copyright [hunk 12]
```diff
@@ -212,6 +212,49 @@ License: public-domain
...
+ images/land/haze2*
+Copyright: Marek Piwnicki
+License: Unsplash
+ Taken from unsplash.com and cropped. This image was
+ uploaded to unsplash.com after June 2017, so it is subject
+ to the Unsplash License. It was also uploaded after February
+ 2018, so it is subject to an additional restriction limiting
+ the sale of unaltered copies.
+ 
+
```
Comment (bene-dictator) [inline copyright:247]:
```suggestion

```
Oopsie

**Comment by bene-dictator** [comment]
Oh my, sorry Doom. I must be feeling out of it today.

**Comment by ziproot** [comment]
Do you have the source for the Unsplash images?

**Comment by bene-dictator** [comment]
I should do, I can get them to you later today.

**Comment by bene-dictator** [comment]
haze0:
- https://unsplash.com/photos/silhouette-of-mountains-under-cloudy-sky-during-daytime-Bllm3LiwFxs

haze1:
- https://unsplash.com/photos/black-and-white-mountains-under-white-sky-during-daytime-NuBvAE6VfSM

haze2:
- https://unsplash.com/photos/the-sun-is-setting-over-a-mountain-range-S2HCRN7D7Ts

snow23:
- https://unsplash.com/photos/snow-covered-mountain-under-blue-sky-during-daytime-Dd2OENYz8H0

File: copyright [hunk 13]
```diff
@@ -212,6 +212,36 @@ License: public-domain
  for the MESSENGER mission. Because it was produced for
  a NASA mission, it is in the public domain.
 
+Files:
+ images/land/haze0*
+Copyright: Izaac Hewitt-Lawrence
+License: Unsplash
+ Taken from unsplash.com and cropped. This image was
```
Comment (ziproot) [inline copyright:219]:
```suggestion
 Taken from https://archive.is/LZQbZ and cropped. This image was
```

File: copyright [hunk 14]
```diff
@@ -212,6 +212,36 @@ License: public-domain
...
+ uploaded to unsplash.com after June 2017, so it is subject
+ to the Unsplash License. It was also uploaded after February
+ 2018, so it is subject to an additional restriction limiting
+ the sale of unaltered copies.
+
+Files:
+ images/land/haze1*
+Copyright: Tobias Rademacher
+License: Unsplash
+ Taken from unsplash.com and cropped. This image was
```
Comment (ziproot) [inline copyright:229]:
```suggestion
 Taken from https://archive.is/wip/Rc9fs and cropped. This image was
```

File: copyright [hunk 15]
```diff
@@ -212,6 +212,36 @@ License: public-domain
...
+ uploaded to unsplash.com after June 2017, so it is subject
+ to the Unsplash License. It was also uploaded after February
+ 2018, so it is subject to an additional restriction limiting
+ the sale of unaltered copies.
+
+Files:
+ images/land/haze2*
+Copyright: Marek Piwnicki
+License: Unsplash
+ Taken from unsplash.com and cropped. This image was
```
Comment (ziproot) [inline copyright:239]:
```suggestion
 Taken from https://archive.is/iVNGc and cropped. This image was
```

File: copyright [hunk 16]
```diff
@@ -2230,6 +2260,16 @@ Copyright: Various
 License: CC0
 Comment: Taken from unsplash.com and cropped.
 
+Files:
+ images/land/snow23*
+Copyright: Daniel Frank
+License: Unsplash
+ Taken from unsplash.com and cropped. This image was
```
Comment (ziproot) [inline copyright:2267]:
```suggestion
 Taken from https://archive.is/xK5Wu and cropped. This image was
```

**Comment by ziproot** [comment]
I just realized that snow23 is a duplicate of iceplains3

**Comment by bene-dictator** [comment]
Iceplains3 is never used, so we should be fine for implementation. But I guess that has some impact on copyright.

**Comment by ziproot** [comment]
If iceplains3 isn’t used, it could just be removed.

**Comment by bene-dictator** [comment]
I think most of the scenes are unused, we can leave that for another PR.

**Review by LorenzoBolla — APPROVED** [review]
Lovely stuff!

**Review by bene-dictator — APPROVED** [review]
ziproot didn't provide a link for snow23 (though perhaps because a similar image exists ingame). Everything else looks alright.

**Comment by ziproot** [comment]
> ziproot didn't provide a link for snow23 (though perhaps because a similar image exists ingame). Everything else looks alright.

The link would be in PR #9459, which was just merged. As the same image is used in both cases, I think the two files should be combined into one entry, as follows:

```
Files:
 images/land/snow23*
 images/scene/iceplains3*
Copyright: Daniel Frank
License: Unsplash
Comment:
 Taken from https://archive.is/xK5Wu. This image was
 uploaded to unsplash.com after June 2017, so it is subject
 to the Unsplash License. It was also uploaded after
 February 2018, so it is subject to an additional restriction
 limiting the sale of unaltered copies.
```

The entry would have to be below images/sceneiceplains*, though.

File: copyright [hunk 17]
```diff
@@ -3422,6 +3441,18 @@ Comment:
...
+ images/land/snow23*
+ images/scene/iceplains3*
+Copyright: Daniel Frank
+License: Unsplash
+Comment:
+ Taken from https://archive.is/xK5Wu. This image was
+ uploaded to unsplash.com after June 2017, so it is subject
+ to the Unsplash License. It was also uploaded after
+ February 2018, so it is subject to an additional restriction
+ limiting the sale of unaltered copies.
```
Comment (ziproot) [inline copyright:3454]:
```suggestion
 Taken from https://archive.is/xK5Wu. These images were
 uploaded to unsplash.com after June 2017, so they are 
 subject to the Unsplash License. They were also uploaded
 after February 2018, so they are subject to an additional
 restriction limiting the sale of unaltered copies.
```

**Review by ziproot — APPROVED** [review]

File: data/map planets.txt [hunk 18]
```diff
@@ -2932,6 +2943,13 @@ planet Nepheritis
 	description `	While these diamonds are unfortunately not economical to collect with current human technology, if you were so inclined you could perhaps make an attempt at catching one as it fell, deep within the atmosphere where they are largest.`
 	government Uninhabited
 
+planet Neriene
+	attributes uninhabited
+	landscape land/haze1
+	description `While lacking its iconic ferrous-red color, this planet is almost eerily similar in conditions to the second planet humanity set foot on, Mars. The thin atmosphere of carbon dioxide, the polar ice caps, and the echoes of dried waterways across the surface all tell the tragic tale of the same fate that befell Mars in aeons past.`
```
Comment (Amazinite) [inline data/map planets.txt:2949]: "Almost" feels redundant/superfluous.
```suggestion
	description `While lacking its iconic ferrous-red color, this planet is eerily similar in conditions to the second planet humanity set foot on, Mars. The thin atmosphere of carbon dioxide, the polar ice caps, and the echoes of dried waterways across the surface all tell the tragic tale of the same fate that befell Mars in aeons past.`
```

File: data/map planets.txt [hunk 19]
```diff
@@ -2932,6 +2943,13 @@ planet Nepheritis
 	description `	While these diamonds are unfortunately not economical to collect with current human technology, if you were so inclined you could perhaps make an attempt at catching one as it fell, deep within the atmosphere where they are largest.`
 	government Uninhabited
 
+planet Neriene
+	attributes uninhabited
+	landscape land/haze1
+	description `While lacking its iconic ferrous-red color, this planet is eerily similar in conditions to the second planet humanity set foot on, Mars. The thin atmosphere of carbon dioxide, the polar ice caps, and the echoes of dried waterways across the surface all tell the tragic tale of the same fate that befell Mars in aeons past.`
```
Comment (Amazinite) [inline data/map planets.txt:2949]:
```suggestion
	description `While lacking its iconic ferrous-red color, this planet is eerily similar in conditions to the second planet humanity set foot on, Mars. The thin atmosphere of carbon dioxide, the polar ice caps, and the echoes of dried waterways across the surface all tell the tragic tale of the same fate that befell Mars in eons past.`
```
